packages/block-library/src/cover/edit.js

```diff
@@ -264,6 +266,47 @@ function CoverEdit( {
 	const onSelectMedia = attributesFromMedia( setAttributes );
 	const isBlogUrl = isBlobURL( url );
 
+	const [ prevMinHeightValue, setPrevMinHeightValue ] = useState( minHeight );
+	const [ prevMinHeightUnit, setPrevMinHeightUnit ] = useState(
+		minHeightUnit
+	);
+	const isMinFullHeight = minHeightUnit === 'vh' && minHeight === 100;
+
+	// Remember previous values of height,
+	// to be able to implement Full Height toggle.
+	useEffect( () => {
```

Comment:
Would it work to each time we execute toggleMinFullHeight, and isMinFullHeight was not true we store the previous value inside toggleMinFullHeight removing the need for an effect ?

Reply:
let's try it Sounds smart to me :-)